nuke-from-orbit/loadtester

```diff
@@ -299,6 +308,27 @@ set_iap_bindings() {
   done
 }
 
+compare_tags() {
+  current_tag=$(kubectl get deployment lw-pod -o json | jq -r '.spec.template.spec.containers[0].image' | cut -f 2 -d ':')
```

Comment:
No way the container could ever get an index other than 0, right? I guess if someone was screwing around making manual changes.

Reply:
yes, they'd have to alter things manually, at which point the benefit of this deployment tool is probably lost on them anyways :)